gui/src/app/pages/HomePage/SaveProjectWindow.tsx

```diff
@@ -211,4 +228,60 @@ const makeSPShareableLinkFromGistUrl = (gistUrl: string) => {
   return url;
 };
 
+type SaveToBrowserViewProps = {
+  fileManifest: Partial<FileRegistry>;
+  title: string;
+  onClose: () => void;
```

Comment:
I know we have this elsewhere in this file too--what's the intended use case? As far as I can see, it's just hard-coded to a no-op?

Reply:
I changed the name to `onCancel` which should be clearer. This is called if user clicks the CANCEL button.